### DISCUSSION TITLE
Can double encrypting a KDBX file corrupt it?

### DISCUSSION THREAD

**Discussion by 1artist** [keepassxreboot/keepassxc#9866] [Q&A] [answered]
Cryptomator encrypts data for secure keeping using an encryption method that is not the same encryption method KDBX uses. In short, if I store a KDBX file inside of Cryptomator I am essentially double encrypting my KDBX file which is already encrypted using a different encryption method. Can double encrypting a KDBX file in this way possibly corrupt the KDBX file over time?  Alternatively, is the KDBX encryption method already strong enough to not even require the need to use Cryptomator as a second encryption layer? I hope somebody can clarify for me if double encrypting a KDBX file is safe on the file itself without having to worry about it becoming corrupt.  Any info welcome and thank you for your time.

**Comment by phoerious** [comment, marked as answer]
If something is corrupted, you will noticed, because it will prevent you from opening the database. But storing the KDBX in an encrypted container should not cause that any more than storing it elsewhere.

**Reply by 1artist** [reply]
Phoerious, thank you for your post. Do you think the double encryption is overkill?  In other words, is the encryption method KDBX uses by default already is secure enough?

**Reply by droidmonkey** [reply]
Yes, absolutely unnecessary to double encrypt
